stdlib/task/src/main/ballerina/src/task/scheduler.bal

```diff
@@ -54,7 +54,7 @@ public type Scheduler object {
 
     # Starts running the task. Task Scheduler will not run until this has been called.
     #
-    # + return - Returns `task:SchedulerError` if the process failed due to any reason, nil otherwise.
+    # + return - `task:SchedulerError` if the process failed due to any reason, nil otherwise
```

Comment:
```suggestion
    # + return - `task:SchedulerError` if the process failed due to any reason, nil otherwise
```
Same as above. Also, please refer the template for the correct format of the return statement description.